### PR TITLE
Add WebP Format Support

### PR DESCRIPTION
**Pull request by sadok-f**
#32

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,8 +143,8 @@ default_options:
   extent: null
   preserve-aspect-ratio: 1
   preserve-natural-size: 1
-  # private options set here
-  thread: 1
+  webp-support: 1
+  webp-lossless: 1
```
Comment (baamenabar) [inline README.md]: lossless should default to 0

File: config/parameters.yml [hunk 2]
```diff
@@ -1,10 +1,10 @@
 application_name: Flyimg.io
 
 #debug
-debug: false
+debug: true
```
Comment (sadok-f) [inline config/parameters.yml]: Yes, I would have it on by default, I assume it's useful especially when first time you setup the application and get an error, so at least you understand why you get it.

File: config/parameters.yml [hunk 3]
```diff
@@ -78,5 +80,7 @@ default_options:
   extent: null
   preserve-aspect-ratio: 1
   preserve-natural-size: 1
+  webp-support: 1
+  webp-lossless: 1
```
Comment (baamenabar) [inline config/parameters.yml]: why is lossless on by default? it should be 0 by default.

File: src/Core/Controller/CoreController.php [hunk 4]
```diff
@@ -29,7 +31,20 @@ public function setApp(Application $app)
...
+        $image->unlinkUsedFiles();
+        return $response;
+    }
+
+    /**
+     * @param Image $image
+     * @param Response $response
+     * @return Response
+     */
+    protected function setHeadersContent(Image $image, Response $response)
```
Comment (baamenabar) [inline src/Core/Controller/CoreController.php]: I liked this separation

File: config/parameters.yml (hunk 3 above)
Comment (sadok-f) [inline config/parameters.yml]: yes true, I'll change that.

File: config/parameters.yml [hunk 5]
```diff
@@ -1,10 +1,10 @@
 application_name: Flyimg.io
 
 #debug
-debug: false
+debug: true
 
-#path to mozjpeg bin
-mozjpeg_path: /opt/mozjpeg/bin/cjpeg
```
Comment (sadok-f) [inline config/parameters.yml]: not need it anymore, I added the path to Image class as const;

**Pull request merged**